build.sh

```diff
@@ -9,20 +9,14 @@ then
   PRERELEASETAG="$PrereleaseTag"
 fi
 
-if [ $(dotnet --info | grep "OS Platform" | grep -c Windows) -ne 0 ]
-then
-  OS=Windows
-else
-  OS=Linux
-fi
-
-FIND=/usr/bin/find
+# Clean up previous builds
+rm -rf {src,test,testing}/*/bin {src,test,testing}/*/obj
 
 CONFIG=Release
 # Arguments to use for all build-related commands (build, pack)
 DOTNET_BUILD_ARGS="-c $CONFIG"
 # Arguments to use for test-related commands (test)
-DOTNET_TEST_ARGS="$DOTNET_BUILD_ARGS"
+DOTNET_TEST_ARGS="--no-build $DOTNET_BUILD_ARGS"
```

Comment:
Is no-build necessary? Doesn't it figure out itself that it doesn't need to re-build?